samples/interactive-tutorials/prediction/symple_prediction_py.md

```diff
@@ -0,0 +1,109 @@
+    - `labels`—the labels applied to a resource. More about the labels could be found in the [documentation](https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements).
+   
+//TODO check the file path
+1. Open <walkthrough-editor-select-regex filePath="cloudshell_open/python-retail/samples/interactive-tutorials/prediction/get_prediction.py" regex="# get prodciction request">prediction/get_prediction.py</walkthrough-editor-select-regex> file and check the `PredictRequest` request.
+Here is a simple prediction request with only required fields set.
+
+1. To send this simplest PredictRequest to the Retail API, run the code sample:
+
+    ```bash
+    python predict/get_prediction.py
+    ```
+
+1. Check the response in the Terminal. The PredictionResult consists of the following fields:
+   -`results[]`-a list of recommended products. The products are ordered by relevance - from the most relevant product to the least.
+   -`attribution_token`-a unique attribution token. The attribute which is used to track the recommendation model performance.
+   -`missing_ids[]`-IDs of products in the request that were missing from the inventory.
+   
+## Return products or scores
+
+You can control either the response will contain the product objects or only the products scores. There are two parameters in the `PredictionRequest` to define that:
+    -`returnProduct`-If set to true, the associated product object will be returned in the `results.metadata` field in the prediction response.
+    -`returnScore`-If set to true, the prediction 'score' corresponding to each returned product will be set in the `results.metadata` field in the prediction response. 
+                    The given 'score' indicates the probability of a product being clicked/purchased given the user's context and history.
+
+Set the `returnScore` parameter, run the same code sample again and check the response:
+    ```python
+    params.returnScore = True
+    ```
+Now the response contains only the probability score, like this:
+
+//TODO add the example of a response
+
+Next, add the `returnProduct` parameter to the `ProdictRequest`:
+    ```python
+    params.returnProduct = True
+    ```
+Run the same code sample and check the product objects are returned.
+    ```bash
+    python predict/get_prediction.py
+    ```
+## Error handling
+
+There are required fields in a PredictRequest:
+    -`placement`
+    -`user_event`
+
+If any of these fields is set with invalid value, the error message is expected. 
+Try to request the Prediction service with non-existent serving config as a placement, change this field in the request in the code sample:
+    ```python
+    placement=f"projects/{project_id}/locations/global/catalogs/default_catalog/placements/nonexistent"
+    ```
+Run the code sample and check the following error message appeared:
+
+//TODO - add the error message
+
+The service returns an error also if the user event passed in the request is invalid.
```

Comment:
The service also returns an error when the user event in the request is invalid.